pkg/app/pipedv1/plugin/inputs.go

```diff
@@ -0,0 +1,24 @@
+package plugin
+
+import (
+	"os/exec"
+
+	"github.com/pipe-cd/pipecd/pkg/app/pipedv1/deploysource"
+	"github.com/pipe-cd/pipecd/pkg/git"
+	"github.com/pipe-cd/pipecd/pkg/plugin/api/v1alpha1/platform"
+)
+
+func GetPlanSourceCloner(input *platform.PlanPluginInput) (deploysource.SourceCloner, error) {
```

Comment:
🤔 I wonder where we should place utilities like this function.
I place it here because all plugins may need this utility, but should I create another package under the plugin package?

Reply:
I got your point, let's find out later. In this PR, place it here is LGTM 👍